<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New</class>
 <widget class="QDialog" name="New">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>213</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Mount Point:</string>
         </property>
         <property name="buddy">
          <cstring>mntPoint</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mntPoint">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/boot/efi</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/home</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/usr/local</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/opt</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>File System &amp;Type:</string>
         </property>
         <property name="buddy">
          <cstring>fsType</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fsType">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>ext2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ext3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ext4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>fat32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>linux-swap</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Size(MB):</string>
         </property>
         <property name="buddy">
          <cstring>fixedSize</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fixedSize">
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="correctionMode">
          <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
         <property name="value">
          <number>5000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="chboxUseAll">
         <property name="text">
          <string>Use All Available Space</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>New</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>New</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
